doc/source/example_notebooks/IS2_data_read-in.ipynb

```diff
@@ -63,9 +63,8 @@
    "metadata": {},
```

Comment:
<p>Few minor typos in the bullets. First bullet: extra word "files". Third bullet: ATL would be a dir name, not filename.</p><p> </p><p>Could we add a bullet that will show getting a specific set of ATL files. Something like: <code>/this/path/*ATL07*.h5</code>: refers to all <code>.h5</code>  files in the <code>/this/path</code> folder that have <code>ATL07</code> in the filename.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icesat2py/icepyx/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>